src/core_landice/mode_forward/mpas_li_thermal.F

```diff
@@ -2673,9 +2674,15 @@ subroutine enthalpy_matrix_elements(&
            harmonic_avg  = .false.  ! if true, take the harmonic average of diffusivity in adjacent layers
                                     ! if false, take the arithmetic average
 
-      tempDiffusivityFactor = 1.0e-5_RKIND
-      ! TZ: this value is somehow empirical, and is picked according to Kleiner's benchemark B. 
-      ! In Greve and Blatter (2016) 'Comparison of thermodynamics solvers in the polythermal ice sheet model SICOPOLIS', it was 1.0e-3
+      character(len=RKIND), pointer :: &
```

Comment:
@tongzhangice , this should be type real, not character.

Reply:
It's real on my side...